views/index.erb

```diff
@@ -26,6 +26,7 @@
       </ul>
       <a href="http://<%= request.host %>:9090" target="_blank">Presentation</a>
       <a href="https://<%= request.host %>" target="_blank">PE Console</a>
+      <a href="https://<%= request.host %>:3000" target="_blank">Code Repository</a>            
```

Comment:
This needs to be gated on the plugin like https://github.com/puppetlabs/puppetfactory/commit/f45eb6d297af8460eab5aaec8496485a2960af48#diff-de7b4f80f1e6de6f4a4be82c477c2f94R29